structure/0072-legal-team.md

```diff
@@ -0,0 +1,88 @@
+# RFC 4269: Legal Team
+
+This document has been made according to the process outlined in [RFC 0006](https://github.com/QuiltMC/rfcs/blob/main/structure/0006-governance.md#teams).
+
+# Summary
+
+The Legal Team is a standalone team in charge of handling legal matters to the best of its ability. This team will not require its members to be lawyers 
+or have any form of training, although this would be preferred. 
+
+# Motivation
+
+Historically, legal matters have been handled by various different people without any proper structure. The de-factor team who would handle particular 
+matters (eg. the Admin Board for registering Quilt as a non-profit) can feel unqualified to handle those issues and/or not have the bandwidth for it. 
+Moreover, someone handling legal matters leaving that de-facto team should not lead to them not handling those matters anymore, especially 
+considering the current scarcity of trained or knowledgeable members.
+
+Some laws may also require us to have a designated legal team, such as to designate a Data Protection Officer 
+([Art. 37 EU GDPR](https://gdpr-info.eu/art-37-gdpr/)).
+
+# Explanation
+
+## Responsibilities
+
+The team is responsible for the following:
+
+- Ensuring Quilt stays up to date with data regulation laws (eg. EU GDPR, COPPA)
+    - Keep the privacy policies, and terms of services up to date
+    - Designate a DPO
+    - Responding to data privacy requests
+    - Assisting the Infrastructure team in staying compliant
+- Reviewing contracts signed by the organisation
+- Handling registration of the organisation as a legal entity, if decided
+- Advising other teams on any legal issues that may arise
+- Answering questions and concerns from community members to the best of their ability
+- Writing guidelines on what changes to Quilt projects require legal verification and providing a process for this verification to happen
+- Write internal or public processes on how to handle requests
+- Any other legal matter not explicitly listed in this document
+
+## Legal Team Server
+
+To perform their activity, the team has a private server not viewable by any other team, for discussing problems needing the uttermost privacy. 
+
+While this deviates from the usual observable approach that makes sure every team can be monitored by at least one other independent team, 
+privacy is legally required to handle some matters, such as data privacy requests.
```

Comment:
We should discuss how we regulate this and assign users to it. Maybe this is also an infra team job? Potentially we need a separate infra team account for this? Or, perhaps, this should still be keyholder-relevant?

That discussion should be detailed here.

Alternatively, if that is not possible, we should discuss outsourcing this team to another organization somehow.

Reply:
We cannot legally justify having more people than strictly necessary seeing sensitive data. An overseeing body isn't strictly necessary according to the law. Usage of that server will be limited to only when legally required.